go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/aukilabs/go-tooling v0.14.2
-	github.com/aukilabs/hagall-common v0.1.4
+	github.com/aukilabs/hagall-common v0.1.5-0.20240313063642-e1d046a9b16c
```

Comment:
Please update the module after a new version is released.

Reply:
updated to v0.1.5